CandidateReranking.md

```diff
@@ -74,27 +72,27 @@ Entites returned from the SemanticSearchAPI.
 
 | Param    | Type | Description
 |----------|------|--------|
-| name | String | **Required**: Name of the keyword, e.g. "Java".
-| type | String | **Required**: Type of the keyword, e.g. "skill".
-| weight | Integer | **Required**: Weight of the keyword as an int.
+| name | String | **Optional**: Name of the keyword, e.g. "Java".
+| type | String | **Optional**: Type of the keyword, e.g. "skill".
+| weight | Integer | **Optional**: Weight of the keyword as an int.
```

Comment:
what are people supposed to use here? is there an acceptable range, like [1, 100]?